core/rio/api/src/main/java/org/eclipse/rdf4j/rio/helpers/SimpleParseLocationListener.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2015 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+package org.eclipse.rdf4j.rio.helpers;
+
+import org.eclipse.rdf4j.rio.ParseLocationListener;
+
+/**
+ * A simple implementation of {@link ParseLocationListener}.
+ * 
+ * @author Peter Ansell
+ */
+public class SimpleParseLocationListener implements ParseLocationListener {
```

Comment:
I have never used ParseLocationListener in the past and don't expect to do so in the future. 

The reasoning for introducing an implementation is simply that we have an interface that we do not plan to deprecate or remove, but we have no local implementation of it, particularly as it is this simple to implement and doesn't add any maintenance costs to the project to do so.